spec/features/social_media_meta_tags_spec.rb

```diff
@@ -41,12 +45,12 @@
       expect(page).to have_css 'meta[name="twitter:site"][content="' + twitter_handle + '"]', visible: false
       expect(page).to have_css 'meta[name="twitter:title"][content="' + meta_title + '"]', visible: false
       expect(page).to have_css 'meta[name="twitter:description"][content="' + sanitized_truncated_meta_description + '"]', visible: false
```

Comment:
I'm not sure checking a sanitized description makes sense anymore :thinking:, since we don't use HTML here. And since we use custom content, maybe we could remove that part.
```suggestion
      expect(page).to have_css 'meta[name="twitter:description"]', visible: false
```